libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/ActiveGuidanceScreen.kt

```diff
@@ -100,8 +106,8 @@ class ActiveGuidanceScreen(
         val mapboxCarMapSurface = carActiveGuidanceContext.mapboxCarMap.carMapSurface
             ?: return@RoutesObserver
         val featureCollection = FeatureCollection.fromFeature(Feature.fromGeometry(coordinate))
-        mapboxCarMapSurface.mapSurface.getMapboxMap().getStyle { style ->
-            placesLayerUtil.updatePlacesListOnMapLayer(style, featureCollection)
+        mapboxCarMapSurface.getStyle {
+            placesLayerUtil.updatePlacesListOnMapLayer(it, featureCollection)
         }
```

Comment:
Will we redraw the points after style change?

Reply:
I've tested it with going to settings, changing a day/night and returning to the app. Destination pin is drawn